appcelerator/titanium_mobile
# [TIMOB-15061] iOS: Adding support for the new naming convention in Xcode 5's retina sim

**Testing instruction** 

Should be able to properly switch between 4inch and 3.5 inch iphone sim and ipad retina properly using studio. 
follow additional instructions in [JIRA](https://jira.appcelerator.org/browse/TIMOB-15061)

Side note : Do test to Make sure this still works fine with Xcode 4.6.2

Please do the code review on https://github.com/appcelerator/ios-sim/pull/4
